### PR TITLE
fix eslint warnings + travis

### PR DESCRIPTION
**Pull request by georgehrke**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -35,6 +36,6 @@ module.exports = {
 		// disable the recommended no-extra-semi
 		semi: ['error', 'always'],
 		// trailing commas
-		'comma-dangle': ['error', 'always']
+		'comma-dangle': ['error', 'never']
```
Comment (skjnldsv) [inline .eslintrc.js]: you can directly remove the line :)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+language: node_js
+addons:
+  firefox: latest-esr
+script: npm run test
```
Comment (skjnldsv) [inline .travis.yml]: https://github.com/nextcloud/vue-components/blob/master/.travis.yml
Please also add a compile check and a lint check :)

**Comment by skjnldsv** [comment]
Could you set "Travis CI - Pull Request In progress — Build Started" as required for merging? I don't have the perms :wink:

**Comment by georgehrke** [comment]
> Could you set "Travis CI - Pull Request In progress — Build Started" as required for merging? I don't have the perms 😉

Done

**Pull request merged**